packages/lesswrong/components/posts/PostsPage.jsx

```diff
@@ -275,7 +275,7 @@ class PostsPage extends Component {
       const htmlBody = {__html: post.htmlBody}
       let query = location && location.query
       const view = _.clone(router.location.query).view || Comments.getDefaultView(post, currentUser)
-      const description = post.plaintextExcerpt ? post.plaintextExcerpt : (post.body && post.body.substring(300))
+      const description = post.plaintextExcerpt ? post.plaintextExcerpt : (post.body && post.body.substring(0, 300))
```

Comment:
This is the sort of thing which unit tests would catch. At some point I'd like to spend some time making unit tests easy to write and run.

Reply:
Yep, would also love that to happen.